modules/pwa/runtime/pwa-plugin.client.ts

```diff
@@ -1,7 +1,12 @@
 import { useRegisterSW } from 'virtual:pwa-register/vue'
+import type { UnwrapNestedRefs } from 'vue'
+import type { PwaInjection } from './types'
 import { STORAGE_KEY_PWA_HIDE_INSTALL } from '~/constants'
+import type { Plugin } from '#app'
 
-export default defineNuxtPlugin(() => {
+const plugin: Plugin<{
```

Comment:
At least we should add a todo comment here that we need to investigate why the injection isn't working in JetBrains. 🤔

In general I'd rather see us fix the underlying issue (in Nuxt, if there is one), or report it to JetBrains so they can fix it rather than requiring users to manually annotate types.

Reply:
removed